Maple2.File.Ingest/Mapper/ItemMapper.cs

```diff
@@ -82,6 +88,12 @@ public class ItemMapper : TypeMapper<ItemMetadata> {
                     .Select(slot => Enum.Parse<EquipSlot>(slot.name, true))
                     .ToArray(),
                 Mesh: data.ucc.mesh,
+                Life: new ItemMetadataLife(
+                    UsePeriod: data.life.usePeriod,
+                    ExpirationPeriod: expirationTime,
+                    ExpirationType: (ItemExpirationType) data.life.expirationType,
```

Comment:
I feel like we can just do the conversion here at parse time and just store duration in seconds rather than calculating it in `Item.cs`

Basically just store `ExpirationDuration` rather than `ExpirationType`+`ExpirationTypeDuration`